TARGETS

```diff
@@ -1,5 +1,5 @@
 # This file @generated by:
-#$ python3 buckifier/buckify_rocksdb.py
+#$ python3 buckifier/buckify_rocksdb.py '{ "fbcode": { "extra_deps": ["//rocks/object_factory:rocks_object_factory"], "extra_compiler_flags": [ "-DROCKSDB_UNITTESTS_WITH_CUSTOM_OBJECTS_FROM_STATIC_LIBS" ] } }'
```

Comment:
It looks like you're building the release directory TARGETS not development directory TARGETS. See also all the new "_fbcode" targets below